web/src/presence.ts

```diff
@@ -206,7 +207,11 @@ export function set_info(
         // system are common in both situations.
         const person = people.maybe_get_user_by_id(user_id, true);
         if (person === undefined) {
-            if (!(watchdog.suspects_user_is_offline() || reload_state.is_in_progress())) {
+            if (
+                !(watchdog.suspects_user_is_offline() || reload_state.is_in_progress()) &&
+                (settings_data.user_can_access_all_other_users() ||
+                    page_params.can_access_all_users_group_limits_presence)
+            ) {
```

Comment:
I have an alternative proposal here, which is to just remove this logging. I'm pretty sure we've fixed all the actual bugs this was good for finding, and it's basically spam now.

Can be a separate commit that just replaces this blueslip logging (i.e. just having a `continue` in this block) with a comment:

> There are a number of situations where it is expected that we get presence data for a user ID that we do not have in our user database, including when we're offline/reloading (watchdog.suspects_user_is_offline() || reload_state.is_in_progress()), when `CAN_ACCESS_ALL_USERS_GROUP_LIMITS_PRESENCE` is enabled, and whenever presence wins a race with the events system for events regarding a newly created or visible user.